translations/monero-core_de.ts

```diff
@@ -954,22 +954,22 @@
     <message>
         <location filename="../pages/Settings.qml" line="102"/>
         <source>Create view only wallet</source>
-        <translation type="unfinished"></translation>
+        <translation>Schreibgeschütze Wallet erstellen</translation>
```

Comment:
"Schreibgeschützt" is really confusing in this context. View-only means that you can only see incoming transactions but can't spend anything, while "schreibgeschützt" implies that you later cant change the file or that it protects any write process on OS level. I understand the difficulty for a translation and maybe we are stuck with a weird translation, but what about something like "Bankauszug - Wallet" (heavily pointing out that you only see and cant do anything), "Beobachtende Wallet" or "Beschränkte Wallet"?